app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -40,7 +42,9 @@ data class User private constructor(
     var matches: List<String>?,
     var likes: List<String>?,
     var recordingPath: String?,
-    var ageRange: List<Int>?
+    var ageRange: List<Int>?,
+    var deleted: Boolean,
+    var reported: List<String>?
```

Comment:
I think ```reported``` is not a great variable name for a list of Strings. It makes me think of a Boolean so maybe a clearer name would be nice.